src/tools/singlejar/output_jar.h

```diff
@@ -18,6 +18,7 @@
 #include <stdio.h>
 
 #include <cinttypes>
+#include <cstddef>
```

Comment:
_Internal reviewer comment_: Is this change needed?

Reply:
This is a change that I forgot to move to another PR. Removed.

Reply:
_forwarded_